### DISCUSSION TITLE
How to add React game?

### DISCUSSION THREAD

**Discussion by Abhinavchoudhary2005** [sarmadhamdani02/WebGames#93] [Q&A]
So, I have made a Tenzies game using React. The link to play the game is [here](https://tenzies-game-abhinav.vercel.app/). I want to add this game to the webgames website. I need help regarding how to integrate the code. I have already sought some assistance from ChatGPT, but I couldn't solve the problem. If anyone can help, I would greatly appreciate it.

**Comment by Abhinavchoudhary2005** [comment]
code of my website is [here](https://github.com/Abhinavchoudhary2005/Tenzies-Game)

**Reply by sarmadhamdani02** [reply]
Thank you so much for the great work 👏 

When this website was built, I didn't think about games developed on other tools except HTML, CSS, and JS as it was built to support beginners. 🤔 

For now, the best approach to add games built-in tools other than HTML/CSS is to create and add the thumbnail of the game to the [main website](https://webgames02.netlify.app/) along with the name and when the user clicks on that, they will be redirected to the corresponding game website (https://tenzies-game-zeta-nine.vercel.app/ in your case). If you take a look at the code of the [main website](https://webgames02.netlify.app/), you'll see how to implement this.

One more thing, the link you shared is showing error 404, I would love to see the game ❤️, please let me know when you solve this problem.

Regards
